lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -168,6 +175,16 @@ public function setTables($entityTables, $manyToManyTables)
         }
     }
 
+    public function setInflector(Inflector $inflector) : void
+    {
+        $this->inflector = $inflector;
+    }
+
+    public function getInflector() : Inflector
```

Comment:
Shouldn't it be final?
```suggestion
    final public function getInflector() : Inflector
```

Reply:
i removed the getters now